modAionImpl/src/org/aion/zero/impl/sync/TaskImportBlocks.java

```diff
@@ -104,6 +99,9 @@
         this.state = new PeerState(NORMAL, 0L);
     }
 
+    ExecutorService executors =
```

Comment:
Can we make executors an argument in the constructor?  Also, I don't know if using every available CPU for this task is what we want (in any case I don't think it should be decided by this class how many threads there are).

Reply:
I think that's fine, but we can set the thread priority in the executors at the lowest level.

Reply:
another thought, change thread priority between diffent sync mode. if it's under normal node, make priority at highest. otherwise, lowest.

Reply:
That is in a sense what we have there now. 
- `TaskImportBlocks` has the highest priority. 
- Saving the received blocks to disk `TaskStorePendingBlocks` has a slightly lower priority, but still high because it is part of the block importing functionality (i.e. when we reach those blocks we don't want to request them again because the task did not finish storing them). 
- Deleting the used blocks becomes critical only when the database grows very big, so the priority for `TaskDropImportedBlocks` is just slightly above normal.